docs/parameters.yaml

```diff
@@ -2255,6 +2255,13 @@ type: bool
 default: true
 components: ["origin", "cache", "director", "registry"]
 ---
+name: Monitoring.Retention
+description: |+
+  The duration of which Prometheus should retain the monitoring data.
+type: string
```

Comment:
```suggestion
type: duration
```

Reply:
I am not opposed to this, however using the duration type requires that the units are in the following: "ns", "us" (or "µs"), "ms", "s", "m", "h".

Reply:
Oh gosh, prometheus extends time.Duration units. 

Let's either add a new type that defines this new `Prometheus Duration` or set it to `Duration`. I want to be able to check all inputs in the config UI and setting them as string prevents that. 

I slightly prefer just using duration for now, and if admins come back and say that it sucks to convert a month into hours then we can create a new type.

Reply:
Ok for now I'll just change it to `duration`.